src/main/scala/chisel3/util/experimental/getAnnotations.scala

```diff
@@ -0,0 +1,9 @@
+package chisel3.util.experimental
+
+import chisel3.internal.Builder
+import firrtl.AnnotationSeq
+
+object getAnnotations {
```

Comment:
@jackkoenig I move this API to `chisel3.util.experimental`. Maybe a good place here?